lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1141,6 +1141,33 @@ def restart_services(self, services: List[str], wait_time: int = 0):
 
         self.wait_for_restart_to_finish(wait_time)
 
+    def restart_single_service(self, service: str, wait_time: int = 0):
+        """
+        Restart a sigle magmad services.
+
+        Args:
+            service: (str) service name
+            wait_time: (int) max wait time for restart of the services
+        """
+        #
+        # This separate restart function is a result of the interdependencies
+        # mentioned in the `restart_services` fct. Since multiple services/
+        # docker containers are restarted, some race condition between these
+        # restarts and 3GPP spec timers occur. Especially in the docker init
+        # mode, this leads to failing tests in CI due to performance issues.
+        # Since we hard code these intedependent container restarts for testing
+        # only, one test is executed with only a MME restart to keep it green;
+        # Test3485TimerForDefaultBearerWithMmeRestart.
+        #
```

Comment:
Should we move this comment to the document string above? That way it is more easily visible, e.g. if a developer uses the quick documentation lookup feature in an IDE?